<?xml version="1.0" encoding="utf-8"?>

<launch>

    <include file="$(find drone_controller)/launch/main_mixedreality.launch" />

    <node pkg="drone_controller" name="program_controller_node" type="avoidance_scenario_controller.py" output="screen">
    </node>

    <node pkg="drone_controller" name="obstacle_drone_node" type="obstacle_drone.py" output="screen"> 
        <param name="/simulated_ip" type="bool" value="True" />
    </node>

    <node pkg="drone_controller" name="avoidance_node" type="avoidance_navigation.py" output="screen">
    </node> 

    <!-- Launch darknet ros -->
    <include file="$(find darknet_ros)/launch/darknet_ros.launch">
        <arg name="image" default="/mixer/sensors/camera" />
        <arg name="network_param_file" default="$(find darknet_ros)/config/yolov2-tiny.yaml"/>
    </include>


    <node pkg="unity_communication" name="UnityRosConnection2" type="UnityRosConnection.py" output="screen">
        <param name="/port" type="int" value="55555" />
        <param name="/position_topic_name" type="string" value="/ground_truth/uav2/pose" />
    </node>


    
</launch>